packages/atomic/src/components/facets/facet-search/facet-search.tsx

```diff
@@ -144,28 +140,27 @@ export class FacetSearch {
         onKeyDown={(e) => this.combobox.onInputKeydown(e)}
         type="text"
         class={
-          'search-input placeholder-on-background-variant flex-grow outline-none focus:outline-none mx-2'
+          'placeholder-on-background-variant flex-grow outline-none focus:outline-none mx-2'
         }
         placeholder={this.strings.placeholder()}
         value={this.component.facetSearchQuery}
       />
     );
   }
 
+  // TODO: remove show more
```

Comment:
Is this note for a future PR? Or was it intended to be done in this one?

Reply:
Future of my PR, I have a separate story for it